api_tests/collections/test_views.py

```diff
@@ -704,12 +709,15 @@ def test_do_not_return_node_pointers_logged_out(self):
     def test_return_node_pointers_logged_in(self):
         res = self.app.get(self.url, auth=self.user_one.auth)
         res_json = res.json['data']
-        assert_equal(len(res_json), 2)
+        assert_equal(len(res_json), 4)
         assert_equal(res.status_code, 200)
         assert_equal(res.content_type, 'application/vnd.api+json')
         first_embedded = res_json[0]['embeds']['target_node']['data']['id']
         second_embedded = res_json[1]['embeds']['target_node']['data']['id']
-        assert_items_equal([first_embedded, second_embedded], [self.project._id, self.public_project._id])
+        # node_links end point does not handle registrations correctly
```

Comment:
This should be documented on the relevant view endpoint documentation and we should say that using node_links is deprecated in favor of linked_nodes.